src/app/pages/psbt-request/use-psbt-request.tsx

```diff
@@ -41,7 +43,10 @@ export function usePsbtRequest() {
         const tx = getPsbtAsTransaction(payload.hex);
 
         try {
-          const signedTx = await signPsbt({ indexesToSign: signAtIndex, tx });
+          const signedTx = await signPsbt({
```

Comment:
Using a `const` could make it a bit more readable here

```suggestion
         const signAtIndex = getDefaultSigningConfig(hexToBytes(payload.hex));
          const signedTx = await signPsbt({tx,
            indexesToSign: signAtIndex,
          });
```